extra-network/libbpf/autobuild/build

```diff
@@ -0,0 +1,2 @@
+make -C src
+make -C src DESTDIR="${PKGDIR}" LIBSUBDIR=lib install install_headers
```

Comment:
Same as above. Also please wrap long lines...

```
make -C "$SRCDIR"/src \
    DESTDIR ... \
    LIBSUBDIR ...\
    install install_headers
```

I recommend limiting to one parameter per line, this will make later revisions easier - as well as making the code more readable.